io/include/pcl/io/impl/auto_io.hpp

```diff
@@ -41,11 +41,11 @@
 #define PCL_IO_AUTO_IO_IMPL_H_
 
 // #include <pcl/io/file_io.h>
-// #include <pcl/io/boost.h>
 #include <pcl/io/pcd_io.h>
 #include <pcl/io/ply_io.h>
 #include <pcl/io/ifs_io.h>
 // #include <pcl/io/vtk_io.h>
+#include <boost/filesystem.hpp> // for path
```

Comment:
What about the commented lines?

Reply:
Removed. Not sure what they were for initially, but they are obviously not needed now.